crowdgezwitscher/blogs/tests/test_api_views.py

```diff
@@ -0,0 +1,159 @@
+import json
+
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from blogs.models import BlogEntry
+
+
+class BlogAPIViewTests(APITestCase):
+    fixtures = ['blogs_views_testdata.json', 'users_views_testdata', 'events_views_testdata.json',
+                'facebook_views_testdata.json']
```

Comment:
überbleibsel von der kurzen Zeit, in der facebook pages und events von einem blogeintrag referenziert wurden